### PR TITLE
[SceneKit] Update up to Xcode 9.2 Beta 2

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Comment by dalexsoto** [comment]
build hang (known issue) https://github.com/xamarin/maccore/issues/5

File: src/scenekit.cs [hunk 1]
```diff
@@ -2720,6 +2720,10 @@ interface SCNSceneSourceLoading {
 		[Field ("SCNSceneSourceStrictConformanceKey")]
 		NSString StrictConformanceKey { get; }
 		
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.TvOS, 11, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.iOS, 11, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "No longer supported.")]
```
Comment (spouliot) [inline src/scenekit.cs]: There's no need for a custom message. The IDE can handle this with a default, localized, message.

Comment (VincentDondain) [reply src/scenekit.cs]: Oh that's a good point, it can indeed!

**Comment by dalexsoto** [comment]
@spouliot fixed, thanks!

**Pull request merged**